src/main/java/org/trigon/hardware/phoenix6/Phoenix6SignalThread.java

```diff
@@ -25,7 +26,9 @@
 import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.locks.ReentrantLock;
 
-
+/**
+ * Provides an interface for asynchronously reading high-frequency measurements to a set of queues for Phoenix 6. Used for getting values from status signals.
```

Comment:
Bruh

Reply:
?

Reply:
This is supposed to help explain it, not complicate it even more